src/views/Dashboard/GeneticAnalyst/Account/index.vue

```diff
@@ -767,6 +801,10 @@ export default {
       }
     },
 
+    onUnstake() {
+      this.hasActiveOrder ? this.showActiveOrder = true : this.showUnstakeDialog = true
+    },
```

Comment:
As far as I know, the ternary operator is not recommended to set variables like this. Then you should use `if` `else` instead

Reply:
updated